ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java

```diff
@@ -161,6 +172,69 @@ public TaskResult run() {
     }
   }
 
+  private boolean removeInvalidAggregatedAccountAndContainerStats(StatsSnapshot currentStats) throws Exception {
+    Map<String, Map<String, Long>> existingStats = accountStatsStore.queryAggregatedAccountStats();
+    List<Pair<Short, Short>> toBeDeletedAccountAndContainers = new ArrayList<>();
+    for (Map.Entry<String, Map<String, Long>> accountEntry : existingStats.entrySet()) {
+      short accountId = Short.valueOf(accountEntry.getKey());
+      for (String containerIdStr : accountEntry.getValue().keySet()) {
+        short containerId = Short.valueOf(containerIdStr);
```

Comment:
Suggest using `Short.parseShort()`

Reply:
why do we prefer `parseShort` over `valueOf`?

Reply:
`parseShort` returns a primitive `short` while `valueOf` returns `Short` object which has an extra unboxing internally.